sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java

```diff
@@ -72,24 +55,52 @@ public void createSmsClienttWithConnectionString() {
             .buildClient();
     }
 
-    /**
-     * Sample code for creating a sync Communication Identity Client using AAD authentication.
-     *
-     * @return the Communication Identity Client.
-     */
-    public SmsClient createCommunicationIdentityClientWithAAD() {
-        // You can find your endpoint and access key from your resource in the Azure Portal
-        String endpoint = "https://<RESOURCE_NAME>.communication.azure.com";
+    public void sendMessageToOneRecipient(SmsClient smsClient) {
+        //Send an sms to only one phone number
+        String to = "<to-phone-number>";
```

Comment:
Can use a fake phone number, also don't need to create a variable for "to", just define it inline.

Reply:
just a reminder that we're ignoring this as per our convo in teams